app/controllers/admin/import_ect/check_ect_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Admin
+  module ImportECT
+    class CheckECTController < AdminController
+      def edit
+        @pending_induction_submission = find_pending_induction_submission
+        @teacher = Teacher.find_by(trn: @pending_induction_submission.trn)
+      end
+
+      def update
+        @pending_induction_submission = find_pending_induction_submission
+
+        check_ect = Admin::ImportECT::CheckECT.new(
+          pending_induction_submission: @pending_induction_submission
+        )
+
+        if check_ect.begin_import!
```

Comment:
Typically I'd expect methods ending in `!` to do something surprising or destructive or non-normal in some way.

Since we're not using `update!` under the hood, this feels little misleading here.

What do you think?